src/Fixtures/Crawlers.php

```diff
@@ -37,6 +37,7 @@ class Crawlers extends AbstractProvider
         '^Jetty\/',
         '^Mail\/',
         '^Mget',
+        '^MicroMessenger\/',
```

Comment:
Hi,

Thanks for the PR. You need to remove the `^` from this line to get the tests to pass.

Thanks 👍